### PR TITLE
Remove checkbox on flaw table

### PR DESCRIPTION
**Pull request by tboyetteredhat**
# [OSIDB-2488] [Remove checkbox column in issue queue]

## Checklist:

- [x] Linting passed
- [x] Type checks passed
- [x] Tests suite passed
- [x] Commits consolidated
- [x] Changelog updated
- [x] Test cases added/updated
- [x] Jira ticket updated

## Summary:

Remove checkbox column in issue queue

## Considerations:

### CODE REVIEW

**Review by superbuggy — APPROVED** [review]

**Review by C-Valen — APPROVED** [review]

**Comment by d10n** [comment]
We should also remove the check box stuff from acceptance tests, `home_page.py` `flawCheckBox`, etc & usages

File: features/pages/advanced_search_page.py [hunk 1]
```diff
@@ -23,12 +23,12 @@ def __init__(self, driver):
...
         "selectValueList2": ("XPATH", "(//select[@class='form-select'])[2]"),
-        "cve_idText": ("XPATH", "//tr[1]/td[2]/a"),
-        "impactText": ("XPATH", "//tr[1]/td[3]"),
-        "sourceText": ("XPATH", "//tr[1]/td[4]"),
-        "titleText": ("XPATH", "//tr[1]/td[6]"),
-        "workflow_stateText": ("XPATH", "//tr[1]/td[7]"),
-        "ownerText":  ("XPATH", "//tr[1]/td[8]"),
+        "cve_idText": ("XPATH", "//tr[1]/td[1]/a"),
+        "impactText": ("XPATH", "//tr[1]/td[2]"),
+        # "sourceText": ("XPATH", "//tr[1]/td[3]"),  # TODO remove unused
```
Comment (huiwangredhat) [inline features/pages/advanced_search_page.py:28]: Removing the source locator is ok since the source field has been removed from the flawlist.

**Review by huiwangredhat — APPROVED** [review]
LGTM.

**Review by RichardXuan — APPROVED** [review]
Automation test case change LGTM

**Review by MrMarble — APPROVED** [review]

**Pull request merged**